src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/admin/controllers/AdminController.java

```diff
@@ -273,14 +278,23 @@ PurgeItemResponse purgeComment(@Valid @RequestBody final PurgeComment purgeComme
 
     final Comment commentToPurge = commentRepository.getReferenceById(
         (long) purgeCommentForm.comment_id());
+
+    logger.info("Purging comment: {}", commentToPurge.getId());
+    logger.info("Purge reason: {}", purgeCommentForm.reason());
+
     try {
-      final int removedCommentHistory = commentHistoryService.deleteAllByComment(commentToPurge);
+      if (commentHistoryConfig.isKeepCommentHistory()) {
+        final int commentHistoryDeleted = commentHistoryService.deleteAllByComment(commentToPurge);
+        logger.info("Successfully deleted {} comments from commentHistory", commentHistoryDeleted);
+      }
```

Comment:
Sorry to review it again, but the same here, that it should ignore it. But the rest is ok.